scripts/sts/sts.sh

```diff
@@ -42,17 +42,26 @@ MINIMAL_POLICY_NAME="${ROLE_NAME}_minimal_policy"
 FUNCTIONAL_TEST_ROLE_NAME="${FUNCTIONAL_TEST_ROLE_NAME:-functional_test_role}"
 FUNCTIONAL_TEST_MINIMAL_POLICY_NAME="${FUNCTIONAL_TEST_ROLE_NAME}_minimal_policy"
 OCM_ENV="${OCM_ENV:-staging}"
-CLUSTER_NAME="${CLUSTER_NAME:-defaultsts}"
+CLUSTER_NAME="${CLUSTER_NAME:-default}}"
 AWS_REGION="${AWS_REGION:-eu-west-1}"
 PREFIX="${PREFIX:-ManagedOpenShift}"
 NODES_COUNT="${NODES_COUNT:-4}"
 MACHINE_TYPE="${MACHINE_TYPE:-m5.xlarge}"
+ENABLE_AUTOSCALING="${ENABLE_AUTOSCALING:-false}"
+MIN_REPLICAS="${MIN_REPLICAS:-4}"
+MAX_REPLICAS="${MAX_REPLICAS:-6}"
+STS_ENABLED="${STS_ENABLED:-true}"
+
 
 provision_sts_cluster() {
     rosa login --env=$OCM_ENV
     rosa create account-roles --mode auto -y
     sleep 30s
```

Comment:
```suggestion
    sleep 30
```

This is a syntax error that I meant to fix. Can you include it with your changes, please?

Reply:
30s seems good, and 30 also is good for sleep in bash ..